packages/fileeditor-extension/src/commands.ts

```diff
@@ -567,7 +567,11 @@ export namespace Commands {
         if (name && widget) {
           const spec = languages.findByName(name);
           if (spec) {
-            widget.content.model.mimeType = spec.mime as string;
+            if (Array.isArray(spec.mime)) {
+              widget.content.model.mimeType = spec.mime[0] as string;
```

Comment:
I think having an empty array would be unlikely but if it happens we could fallback to `text/plain`. What do you think?

Reply:
seems totally reasonable.